src/networking.c

```diff
@@ -537,12 +537,18 @@ void addReplyError(client *c, const char *err) {
     afterErrorReply(c,err,strlen(err));
 }
 
+void addReplyErrorSdsExt(client *c, sds err, int flags) {
```

Comment:
let's add some top comment listing the flags.
nit pick: i think `Ex` suffix is nicer.

Reply:
Fixed